website/docs/theme-classic.md

```diff
@@ -154,3 +154,20 @@ module.exports = {
   },
 };
 ```
+
+### Adding support of new languages
```

Comment:
I think we should document these under the "Syntax highlighting" section of "Markdown Features" instead.